en.txt

```diff
@@ -0,0 +1,78 @@
+Adomas Daškevičius, Vilnius.
+
+About me:
+	I mostly have experience with Linux servers, Unix, c programming language
+	I like Implementing interesting ideas.
+
+Experience:
+	2024.07 - 2024.09: I worked with a professional mentor who assigned me Java tasks, 
+	which were reviewed, corrected, and completed through a process involving 
+	Jira for task management, Git for version control, and code reviews
+	conducted via Pull Requests. 
+	You can see the entire tasking process in the GitHub links below.
+		https://github.com/Adomo-Software/java-fix-me
+		https://github.com/Adomo-Software/rating-system
+		https://github.com/Adomo-Software/caesars-palace-casino
+		https://github.com/Adomo-Software/cherry-tree-hotel
+		https://github.com/Adomo-Software/continental-hotel
+		https://github.com/Adomo-Software/hotel
+	Jira projects (Tasks):
+		[Java basics: Hotel](https://algirdasalisauskas.atlassian.net/jira/software/projects/JBH/list)
+		[Java Collections](https://algirdasalisauskas.atlassian.net/jira/software/projects/JC/list)
+		[Java Fix Me](https://algirdasalisauskas.atlassian.net/jira/software/projects/JFM/list)
```

Comment:
Not list link, board link is better. 
P.s.: CV in PDF format better to add links as hyperlinks